sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -4,41 +4,73 @@
 
 ```ts
 
+import * as coreClient from '@azure/core-client';
+import * as coreHttps from '@azure/core-rest-pipeline';
 import { OperationOptions } from '@azure/core-client';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PipelineOptions } from '@azure/core-rest-pipeline';
 import { TokenCredential } from '@azure/core-auth';
 
+// @public
+export type ArtifactArchitecture = "386" | "amd64" | "arm" | "arm64" | "mips" | "mipsle" | "mips64" | "mips64le" | "ppc64" | "ppc64le" | "riscv64" | "s390x" | "wasm" | string;
```

Comment:
does this type provide value? TS will collapse this to `type ArtifactArchitecture = string`

Reply:
Changed to string, and export `KnownArtifactArchitecture` literal unions.